cli/args.go

```diff
@@ -110,11 +110,7 @@ func parseTerragruntOptionsFromArgs(args []string) (*options.TerragruntOptions,
 	opts.Logger.SetDefaultConsoleHookLevel(loggingLevel)
 	opts.Logger.SetColor(!util.ListContainsElement(opts.TerraformCliArgs, "-no-color"))
 	if fileLoggingDir != "" {
-		absLoggingFileDir, err := filepath.Abs(fileLoggingDir)
-		if err != nil {
-			return nil, err
-		}
-		opts.Logger.AddFile(filepath.Join(absLoggingFileDir, opts.Logger.GetModule())+".txt", fileLoggingLevel)
+		opts.Logger.AddFile(fileLoggingDir, true, fileLoggingLevel)
```

Comment:
The second argument sets the logger as a folder instead of a file. This means that logs will be written in the `dir/logger.submodule.log` path